docs/src/running-validator/validator-troubleshoot.md

```diff
@@ -15,3 +15,29 @@ testnet participants, [https://discord.gg/pquxPsq](https://discord.gg/pquxPsq).
 - [Core software repo](https://github.com/solana-labs/solana)
 
 Can't find what you're looking for? Send an email to ryan@solana.com or reach out to @rshea\#2622 on Discord.
+
+## Blockstore
+
+The validator blockstore rocksdb database can be inspected using the `ldb` tool.
+`ldb` is part of the `rocksdb` code base and is also available in the `rocksdb-tools`
+package.
+
+## Downgrade
+
+If a new column family has been introduced to the validator blockstore a
+subsequent downgrade of the validator to a version prior to the version where
+the new column family was intoduced will cause the validator to fail during
+startup.
+
+List column families:
+```
+ldb --db=<validator ledger path>/rocksdb/ list_column_families
+```
+
+**Warning**: Please seek guidance on discord before modifying the validator
```

Comment:
i see you didn't volunteer your discord handle here to direct support inquiries! :sweat_smile: